openmls/src/group/mls_group/test_mls_group.rs

```diff
@@ -142,6 +142,7 @@ fn remover(ciphersuite: Ciphersuite, backend: &impl OpenMlsCryptoProvider) {
 
     let queued_messages = alice_group
         .propose_remove_member(backend, &alice_signer, LeafNodeIndex::new(1))
+        .map(|(out, _)| out)
         .expect("Could not propose removal");
```

Comment:
For all the similar ones prefer this:
```suggestion
    let (queued_messages, ..) = alice_group
        .propose_remove_member(backend, &alice_signer, LeafNodeIndex::new(1))
        .expect("Could not propose removal");
```